Makefile

```diff
@@ -237,6 +237,13 @@ build_tools/closure.jar: build_tools/.closure_compiler_version
 	wget -P build_tools https://github.com/mykmelez/closure-compiler/releases/download/$(CLOSURE_COMPILER_VERSION)/closure.jar
 	touch build_tools/closure.jar
 
+build_tools/emsdk-portable.tar.gz:
+	wget -P build_tools https://s3.amazonaws.com/mozilla-games/emscripten/releases/emsdk-portable.tar.gz
```

Comment:
We'd also need a way to update the SDK when we want to use a different version, like we do with the other build tools.